lib/streamlit/runtime/forward_msg_queue.py

```diff
@@ -33,6 +33,14 @@ class ForwardMsgQueue:
     a single thread.
     """
 
+    _before_enqueue_msg: Callable[[ForwardMsg], None] | None = None
+
+    @staticmethod
+    def on_before_enqueue_msg(
```

Comment:
nit: maybe also add a brief docstring comment here and mention that it is used as part of static streamlit app generation.